engine/src/hopeit/app/client.py

```diff
@@ -108,6 +119,7 @@ def app_client(app_connection: str, context: EventContext) -> Client:
 async def app_call(app_connection: str,
                    *, event: str, datatype: Type[EventPayloadType],
                    payload: EventPayload, context: EventContext,
+                   datatypes: Optional[Dict[int, Type[EventPayloadType]]] = None,
```

Comment:
A better name instead of `datatypes` could be `responses` as on open api spec: https://github.com/leosmerling-hopeit/hopeit.engine/blob/ceff8ae5204163f277e96348f754f15be0e14509/apps/examples/simple-example/src/simple_example/query_something.py#L23